charts/nri-kube-events/values.yaml

```diff
@@ -36,6 +36,29 @@ sinks:
   # The newRelicInfra sink sends all events to New relic.
   newRelicInfra: true
 
+# If you wish to provide your own newrelic.yml file include it under config:
+# the sample config file is included here as an example. Note that some option
+# overlap with environment variables: for example proxy
+# config:
```

Comment:
How would this play with `nrStaging`?
https://github.com/newrelic/helm-charts/blob/c876e9fbdb253b9a5ea180c3531c4e2bb4802c41/charts/nri-kube-events/values.yaml#L122

I think we can keep both (for compatibility) but perhaps we could add a comment on which one will take priority.

Reply:
it is like the proxy, environment variable take precedence, i'll add a note for that